app/commands/v2/import.rb

```diff
@@ -0,0 +1,98 @@
+module Commands
+  module V2
+    class Import < BaseCommand
+      def call
+        unless UuidValidator.valid?(payload[:content_id])
+          raise CommandError.new(
+          code: 422,
+          error_details: {
+            error: {
+              code: 422,
+              message: "Content id not valid",
+              fields: "content_id",
+            }
+          })
+        end
+
+        delete_all(payload[:content_id])
+        payload[:content_items].map.with_index do |event, index|
+          create_content_item(event, index, payload[:content_id])
```

Comment:
This is the error you get:
```
     CommandError:
       User facing version draft content item cannot be behind the live content item (1 < 2)
```

I can see a few options here, but I'm hesitant to make any changes while the changes to the database schema are underway.